src/js/claims-status/components/appeals-v2/Issues.jsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import CollapsiblePanel from '../../../common/components/CollapsiblePanel';
+
+const Issues = ({ issues }) => {
+  // TO-DO: think about whether it's crazy to run through the issues list 4 times
+  const open = issues.filter(i => i.status === 'open');
+  const remand = issues.filter(i => i.status === 'remand');
+  const granted = issues.filter(i => i.status === 'granted');
+  const denied = issues.filter(i => i.status === 'denied');
+
+  const getListItems = (item, i) => {
+    return (
+      <li key={`${item}-${i}`}>
```

Comment:
`item` will be an object here, right? Did you mean `item.status`?

Reply:
😬 yep